### DISCUSSION TITLE
Italic Font

### DISCUSSION THREAD

**Discussion by master-of-none** [zed-industries/zed#6663] [Feature Requests]
I love writing code in Zed and have completely switched from VS Code to Zed.
I am using **Victor Mono** Font. Is there a way to enable italic font for things like comments or keywords?

**Comment by nickpapciak** [comment]
late to the party but you can put this in your `~/.config/zed/settings.json` file

```
  "experimental.theme_overrides": {
    "syntax": {
      "comment": {
        "font_style": "italic"
      }
    }
  }
  ```

**Reply by master-of-none** [reply]
DId they add it? 
Lol, I switched to Neovim and sometimes use VS Code.

**Reply by rubjo** [reply]
What a nice font! This was added a few months back. You can also install and use a theme extensions that employs italics, like eg. my Ultimate Dark Neo – and I'm sure some others.

<img width="637" alt="image" src="https://github.com/user-attachments/assets/15b5673c-be0a-418c-b803-627a6d4eec7a">

**Comment by elisiariocouto** [comment]
I'm using M Plus 1 Code (patched with Nerd Fonts) and the italics are not working. I've installed the M Plus 1 Code without the Nerd Fonts patch and it still doesn't work. Using the same font in iTerm2 and VS Code displays italics. Does anyone know how to troubleshoot this?

**Reply by elisiariocouto** [reply]
After a bit of searching, I found that this font does not have Italic variations. VS Code and Sublime apparently italicize fonts that do not have Italic variations. Is there any plans to support this?

**Reply by JulindM** [reply]
What VSCode is doing, is I think that is "oblique"ing the font. The font may not have italics, but it can be obliqued to make it slanted.

**Comment by rahulj51** [comment]
Did this break in the latest release? It was working till recently but has stopped working since 0.163.2

**Reply by rubjo** [reply]
Works for me in 0.164.0 preview, anyway.

**Reply by rahulj51** [reply]
ok. I figured what was wrong. I had set `buffer_font_weight` to 500. Zed is not rendering this for some reason.

**Comment by ao** [comment]
Still the same issue on Zed 0.176.3

**Reply by not-a-cowfr** [reply]
https://github.com/zed-industries/zed/discussions/6663#discussioncomment-10930390
(if you want to override other things check out theme docs or https://zed-themes.com/themes/edit)

if that doesnt work make sure your font actually has an italicized version
